Makefile

```diff
@@ -51,10 +56,12 @@ test: fmt
 .PHONY: server-docker
 server-docker:
 	@docker build -f $(SERVER_DOCKERFILE) -t $(SERVER_IMAGE):$(IMAGE_VERSION) .
+	@docker tag $(SERVER_IMAGE):$(IMAGE_VERSION) $(SERVER_IMAGE):latest
```

Comment:
I like this change, but I think it would be cleaner to have multiple `-t` in the `docker build` command.

```sh
docker build -t some_image:first -t some_image:second .
```